solutions/devsprint-leonardo-santos-3/FinanceAppTests/TestUtils/Fixtures/HomeData+Fixture.swift

```diff
@@ -0,0 +1,11 @@
+@testable import FinanceApp
+
+extension HomeData {
+    static func fixture(
+        balance: Float, savings: Float, spending: Float, activity: [Activity]
```

Comment:
As fixtures são interessantes por já definirem valores padrão para os argumentos que são passados, podemos adicionar? 
```suggestion
        balance: Float = 0.0, savings: Float = 0.0, spending: Float = 0.0, activity: [Activity] = []
```